gtnh-modpack.json

```diff
@@ -1,8 +1,7 @@
 {
   "releases": [
-    "2.1.2.4-pre29",
-    "2.1.2.0",
```

Comment:
what about 2.1.2.0?  Don't we want that for the changelog?

Reply:
oh yeah true, small oversight